docs/documentation/backwards-compatibility.md

```diff
@@ -3,17 +3,16 @@ title: Use backwards compatibility to upgrade from version 6
 ---
 # Use backwards compatibility to upgrade from version 6
 
-Version 7 of the Prototype Kit uses the new GOV.UK Design System. It is not compatible with prototypes built with older versions by default.
+Version 7 and above of the Prototype Kit uses the new GOV.UK Design System. They are not compatible with prototypes built with older versions by default.
 
```

Comment:
`Versions 7 and above of the Prototype Kit use the new GOV.UK Design System. They are not compatible with prototypes built with older versions.`